Game/TestSuite.uproject

```diff
@@ -1,6 +1,6 @@
 {
 	"FileVersion": 3,
-	"EngineAssociation": "{AD63C9CA-4E14-B025-434B-93B2522A7DA4}",
+	"EngineAssociation": "{D1DB1A5F-4490-E7AF-B51A-29A7D9CD18BC}",
```

Comment:
Should we check this file in?

Reply:
Bah, thought I had fixed this. Will remove.

Reply:
Hmm I have committed the fix for this, but github doesn't seem to be updating...